contributing.md

```diff
@@ -222,3 +222,82 @@ When adding documentation for a new component, add the new dependency in `packag
 - Update the [changelog](/changelog.md) with the package versions and update the title with today’s date and a brief summary of what has changed.
 - Commit and push to master.
 - Update the #backpack Slack channel and the [BackpackNeedToKnow](mailto:backpackneedtoknow@skyscanner.net) mailing list if necessary.
+
+## React Native
```

Comment:
This section should come before "Publishing packages" I reckon. Also we should add an item to the table of contents at the top of this file.

Reply:
Done - I put this at the bottom initially as I couldn't decide where it would best fit, and forgot to move it up before making the PR.